### ISSUE TITLE
Some games missing in query nbaPlayer / latestFixtureStats

### ISSUE THREAD

**Issue by PacoASBH** [script-money/sorare-nba-lineup-tool#2]
Hello,

My goal is to list all scores of a NBA Player.

But my Graphql query didn't return games in which my player is out the squad. It's normal but I want to know that he missed this game and it doesn't appear in the query result. Someone have idea to help me please. 

![Screenshot4](https://user-images.githubusercontent.com/121180654/209253655-f44f159c-33eb-4cd1-8b8f-eb7e604e129d.PNG)

Missing games (where Thomson was out the squad) on the 15/12/2022.

![Screenshot5](https://user-images.githubusercontent.com/121180654/209253792-1ba1a5b6-6800-48b0-9273-9a85162415b6.PNG)

Thanks

Pascal

**Comment by script-money** [comment]
https://www.cbssports.com/nba/schedule/20221215/ 
GSW has no game on 15/12/2022.

**Comment by PacoASBH** [comment]
Yes I know :( but I'd like to list all last GSW games and Klay THomson scores in each (even if DNP) like below :
![Screenshot6](https://user-images.githubusercontent.com/121180654/209347389-991f3871-f12b-4ed6-aa1d-31f876b18ea7.PNG)

The API allows to know if the player stays bench but not if he is DNP.
It' important to calculate the L5 and L10 for example.

**Comment by script-money** [comment]
<img width="917" alt="Screenshot 2022-12-25 at 09 09 26" src="https://user-images.githubusercontent.com/74860630/209454398-17dc903f-5209-4ecb-9ae1-d4ef2062f39f.png">

You can use *StatusIconType*.
But this attribute is sometimes unreliable.
